src/MainWindow.vala

```diff
@@ -3,61 +3,57 @@
  * SPDX-FileCopyrightText: 2021 elementary, Inc. (https://elementary.io)
  */
 
-public class Music.MainWindow : Hdy.ApplicationWindow {
+public class Music.MainWindow : Gtk.ApplicationWindow {
```

Comment:
AdwApplicationWindow? Or, set the titlebar to a blank widget. Otherwise we have this: https://imgur.com/zsvezd0

Reply:
FTR setting titlebar to an invisible widget is the only thing AdwApplicationWindow does now. And the only reason it exists.